tools/cs_profile_template.yaml

```diff
@@ -1,12 +0,0 @@
-table:
-
-table_definition:
-
-keyspace:
-
-keyspace_definition:
-
-queries:
-  test_query:
-    cql:
-    fields: samerow
```

Comment:
Hmm, it's gone! So quick! Why was it introduced in the first place then?

Reply:
I'm sorry. I need to rebase and remove the appearance of this file at all. At first, I wanted to generate test files from this template in CI. This is a remnant that I somehow missed. I think it's better to have yaml files for each type in the repo, so it is clear what CQL statements are executed, what keyspaces/tables we use, etc.